.github/actions/inductor-xpu-e2e-test/action.yml

```diff
@@ -158,11 +154,6 @@ runs:
           cat $var >> inductor_log/summary_accuracy.csv
```

Comment:
Why we need this csv file? And if the on-demand test both for accuracy and performance, all csv file will be concat together, for example https://github.com/intel/torch-xpu-ops/actions/runs/9659586596

Reply:
Because want to keep one file for all tests instead of to check one-by-one when finished. I will split the performance and accuracy.

Reply:
Seems we don't need this file anymore? I don't find it has been used in any place